src/main/java/seedu/address/model/application/InterviewSlot.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.model.application;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
```

Comment:
I think you can clear away these extra lines while you are reverting the changes to lchokhoe.png.